src/node.cc

```diff
@@ -1092,8 +1092,13 @@ InitializationResult InitializeOncePerProcess(
     // to be loaded, but the default section in that file will not be used,
     // instead only the section that matches the value of conf_section_name
     // will be read from the default configuration file.
-    //  fprintf(stderr, "appanme: %s\n", conf_section_name);
     const char* conf_file = nullptr;
+    // To allow for using the previous default where the 'openssl_conf' appname
+    // was used, the command line option 'openssl-shared-config' can be used to
+    // force the old behavior.
```

Comment:
In that case, we could at least provide a warning for a potential security vuln?

Reply:
I'm a little concerned that a user specifying this options would hopefully know about the implications of using it, and getting a warning might be annoying in that case.

Reply:
Fair enough. Maybe documentation advice?